app/src/main/java/io/neurolab/activities/ShareDataActivity.java

```diff
@@ -70,6 +75,15 @@ protected void onCreate(Bundle savedInstanceState) {
                 newFileName[i] = files[i].getName();
                 j++;
             }
+            numberOfFiles.setVisibility(View.INVISIBLE);
+            Toast.makeText(this, R.string.share_screen_toast, Toast.LENGTH_SHORT).show();
+        }
+
+        if(appDir.listFiles().length == 0)
+        {
```

Comment:
We use inline brackets

Reply:
> As I can see from the sample, when a user deletes, have to come back to see the no files message. Can you implement it in such a way that, once the last dataset is deleted, the message is shown?

 @CloudyPadmal  The "No data to show" message and DataSet deletion is happening on different activities.
"No data to show" is being displayed in Share data activity whereas dataset Deletion is being done in Data Logger Screen.
Should I add similar code for Data Logger screen as well in this PR only?